op-e2e/actions/l2_proposer_test.go

```diff
@@ -110,16 +109,12 @@ func RunProposerTest(gt *testing.T, deltaTimeOffset *hexutil.Uint64) {
 
 	// check that L1 stored the expected output root
 	if e2eutils.UseFPAC() {
-		optimismPortal2Contract, err := bindingspreview.NewOptimismPortal2(sd.DeploymentsL1.OptimismPortalProxy, miner.EthClient())
-		require.NoError(t, err)
-		respectedGameType, err := optimismPortal2Contract.RespectedGameType(&bind.CallOpts{})
-		require.NoError(t, err)
 		disputeGameFactoryContract, err := bindings.NewDisputeGameFactory(sd.DeploymentsL1.DisputeGameFactoryProxy, miner.EthClient())
 		require.NoError(t, err)
 		gameCount, err := disputeGameFactoryContract.GameCount(&bind.CallOpts{})
 		require.NoError(t, err)
 		require.Greater(t, gameCount.Uint64(), uint64(0), "game count must be greater than 0")
-		latestGames, err := disputeGameFactoryContract.FindLatestGames(&bind.CallOpts{}, respectedGameType, new(big.Int).Sub(gameCount, common.Big1), common.Big1)
+		latestGames, err := disputeGameFactoryContract.FindLatestGames(&bind.CallOpts{}, proposer.driver.Cfg.DisputeGameType, new(big.Int).Sub(gameCount, common.Big1), common.Big1)
```

Comment:
This would be easier to update in the future if there was a top-level `op-e2e` `respectedGameType` variable that was referenced in all the `proposer.driver.Cfg.DisputeGameType` instances instead of manually referencing this external config variable each time.

Reply:
Or an `op-e2e/actions/utils.go` file since it seems to only be used here?

Reply:
Wdyt about https://github.com/ethereum-optimism/optimism/pull/9722/commits/795e5ec220681285657dc790d02b3f16532cd593?

Reply:
That mstm!